stable/anchore-admission-controller/values.yaml

```diff
@@ -79,3 +79,8 @@ credentials: {}
   #   password: password1
   # - username: user2
   #   password: password2
+
+# Define custom environment variables to pass to init-ca pod
+extraEnv: []
```

Comment:
I think to make this more clear that it is specifically for the init-ca pod, it should have something clear in the name. Perhaps either nest it under an 'init-ca' object or `initCaExtraEnv` as the name so that it isn't confusing to folks who might expect it to be in the other pod as well.

Reply:
@zhill Makes sense. I'll do the update.

Reply:
@zhill Done 😀